testcode/lib/conio.c

```diff
@@ -120,6 +132,11 @@ void main(void)
                     cputc(i);
                     inpos = (inpos + 1) & 7;
                 }
+#endif
+/* not all targets have waitvblank() */
```

Comment:
yes, IF you add joystick input (which doesnt make a lot of sense here =P) AND dont check with kbhit() inside the loop (as it should have done from the start in the first place), then what you say is true :)